Library/Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,66 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "http://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "http://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "git://anongit.freedesktop.org/gstreamer/qt-gstreamer"
+
+  option "with-static", "Build QtGStreamer as a static library"
+  option "with-tests", "Build QtGStreamer's tests"
+  option "without-examples", "Build QtGstreamer's examples"
+  option "with-codegen", "Build and use QtGStreamer's codegen"
+  option "with-qt5", "Build against Qt5"
+  option "with-qt-dir", "Add supply CMake with qmake of the Qt installation to"
+         "use. Specify it in the environment variable HOMEBREW_QTGSTREAMER_QT."
+         "Use this options to build against Qt versions not installed with"
+         "homebrew."
```

Comment:
Bummer. Since qt5 conflicts with qt there is no way in homebrew to have a Qt4 and a Qt5 installation in parallel (This is useful if your application needs to work with different Qt versions). Therefore third-party installs would be nice. The only thing homebrew would have to support is passing build flags on to the underlying build system (and I believe this would be useful for other formulae too), i.e.:
`brew install qt-gstreamer --build-from-source -- -DCMAKE_PREFIX_PATH=/third/party/Qt/install`.
`--` is used to split brew options from options to be passed to the build system. Of course that would only work with --build-from-source, not with bottled formulae.
Either way, I will remove --with-qt-dir.